service/src/test/java/uk/gov/hmcts/reform/fpl/model/order/OrderTest.java

```diff
@@ -214,6 +219,10 @@ void checkCoverage() {
             Arguments.of(C43A_SPECIAL_GUARDIANSHIP_ORDER, ORDER_DETAILS, Optional.of(REVIEW)),
             Arguments.of(C43A_SPECIAL_GUARDIANSHIP_ORDER, REVIEW, Optional.of(OTHER_DETAILS)),
             Arguments.of(C43A_SPECIAL_GUARDIANSHIP_ORDER, OTHER_DETAILS, Optional.empty()),
+            Arguments.of(C45A_PARENTAL_RESPONSIBILITY_ORDER, ISSUING_DETAILS, Optional.of(CHILDREN_DETAILS)),
+            Arguments.of(C45A_PARENTAL_RESPONSIBILITY_ORDER, CHILDREN_DETAILS, Optional.of(ORDER_DETAILS)),
+            Arguments.of(C45A_PARENTAL_RESPONSIBILITY_ORDER, ORDER_DETAILS, Optional.of(REVIEW)),
+            Arguments.of(C45A_PARENTAL_RESPONSIBILITY_ORDER, REVIEW, Optional.of(OTHER_DETAILS)),
```

Comment:
missing the last transition from OTHER DETAILS -> emtpy()

Reply:
updated